AirCasting/SessionViews/ABMeasurementsView.swift

```diff
@@ -95,12 +105,18 @@ struct SingleMeasurementView: View {
     
     func showStreamName() -> String {
         guard let streamName = stream.sensorName else { return "" }
-        return streamName
-            .drop { $0 != "-" }
-            .replacingOccurrences(of: "-", with: "")
+        if streamName == "Phone Microphone" {
```

Comment:
this string needs to go to a constant. Reason being - it's used in two completely different places:
* Here
* `MicrophoneManager` - when creating mic sessions

Now imagine a task that would say "we need to rename mic session stream names from "Phone Microphone" to "Microphone". Someone performing this change would just go to `MicrophoneManager` and change the string there accidentally braking this piece of logic and probably wouldn't spot it because it's not easy to spot.

If we introduce a constant for this on the other hand, the developer would go there instead and change this constants value. Now both places would get the change and no bugs would be introduced by accident.